client/v2/autocli/common.go

```diff
@@ -224,11 +226,15 @@ func enhanceCustomCmd(builder *Builder, cmd *cobra.Command, cmdType cmdType, mod
 
 // outOrStdoutFormat formats the output based on the output flag and writes it to the command's output stream.
 func (b *Builder) outOrStdoutFormat(cmd *cobra.Command, out []byte) error {
-	var err error
-	outputType := cmd.Flag(flags.FlagOutput)
+	clientCtx, err := client.GetClientTxContext(cmd)
```

Comment:
Can we directly get the client context from its context key instead of using the helper and if not present fallback to the current logic? We are trying to make client.Context optional in the AutoCLI core logic (I know we are not there yet 😅, but that change would not add extra dependencies)

Reply:
seems still need import client for context and key

Reply:
Yes, we need to import client context anyway for the type check, but that's okay. It just makes clearer what our deps on client.Context are.